pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -270,6 +271,33 @@ func (c *Cluster) startProvisioning(config *provisionConfig) {
 			}
 		}
 	}
+
+	// Deploying test based on the storageclassDeviceSet
+	// Todo: Replace the below logic to deploy osd pods over pvcs
+	for _, storageclassDeviceSet := range c.DesiredStorage.Storage {
+		for i := 0; i < storageclassDeviceSet.Count; i++ {
+			sc := "test-sc"
+			pvc := &v1.PersistentVolumeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "pvc-" + storageclassDeviceSet.Name + fmt.Sprintf("-%v", i),
```

Comment:
The names should be uniquely generated by appending a hash/UUID.

Reply:
yup will do it

Reply:
added the logic to do it